plugins/aws/cloudtrail/cloudtrailBucketAccessLogging.js

```diff
@@ -46,6 +56,13 @@ module.exports = {
                 if (!trail.S3BucketName || (trail.HomeRegion && trail.HomeRegion.toLowerCase() !== region)) return cb();
                 // Skip CloudSploit-managed events bucket
                 if (trail.S3BucketName == helpers.CLOUDSPLOIT_EVENTS_BUCKET) return cb();
+                
+                if (!listBuckets.data.find(bucket => bucket.Name == trail.S3BucketName)) {
+                    helpers.addResult(results, 2,
+                        'Unable to locate S3 bucket, it may has been deleted',
```

Comment:
```suggestion
                        'Unable to locate S3 bucket, it may have been deleted',
```
```suggestion
                        'Unable to locate S3 bucket, it may has been deleted',
```